Sources/Testing/Testing.docc/Traits/Trait.md

```diff
@@ -0,0 +1,32 @@
+# ``Trait``
+
+## Topics
+
+### Enabling and disabling tests
+
+- ``Trait/enabled(if:_:fileID:filePath:line:column:)``
+- ``Trait/enabled(_:fileID:filePath:line:column:_:)``
+- ``Trait/disabled(_:fileID:filePath:line:column:)``
+- ``Trait/disabled(if:_:fileID:filePath:line:column:)``
+- ``Trait/disabled(_:fileID:filePath:line:column:_:)``
+
+### Limiting the running time of tests
+
+- ``Trait/timeLimit(_:)``
+ 
+### Categorizing tests
+
+- ``Trait/tags(_:)-505n9``
+- ``Trait/tags(_:)-yg0i``
+
+### Associating bugs
+
+- ``Trait/bug(_:relationship:_:)-86mmm``
+- ``Trait/bug(_:relationship:_:)-3hsi5``
+
+### Adding comments
+- ``Trait/comments``
```

Comment:
I meant the property _and_ the factory function, sorry!

Reply:
Actually let's just switch it back to the function—I think we need to think about the corresponding properties for _all_ of these traits. Sorry for the noise.

Reply:
Yeah ignore me here. I think I confused myself.